scale/scale/local_settings_dev.py

```diff
@@ -56,3 +56,6 @@
 
 # The full name for the Scale Docker image (without version tag)
 SCALE_DOCKER_IMAGE = 'geoint/scale'
+
+# The location of the config file containing Docker credentials
+CONFIG_URI = os.environ.get('CONFIG_URI', CONFIG_URI)
```

Comment:
We don't generally use env vars in the dev settings. Since there's no reasonable default for this, it should default to settings.py so just remove this change.